install_process.go

```diff
@@ -34,40 +34,25 @@ func NewPipInstallProcess(executable Executable, logger scribe.Emitter) PipInsta
 }
 
 // Execute installs the pip dependencies from workingDir/requirements.txt into
-// the targetPath. The cachePath is used for the pip cache directory. If the
-// vendor directory is present, the pip install command will install from local
-// packages.
+// the targetPath. The cachePath is used for the pip cache directory.
+//
+// The pip install command will install from local packages if they are found at
+// the directory specified by `BP_PIP_DEST_PATH`, which defaults to `vendor`.
 func (p PipInstallProcess) Execute(workingDir, targetPath, cachePath string) error {
+	vendorDir := filepath.Join(workingDir, "vendor")
+
+	if destPath, exists := os.LookupEnv("BP_PIP_DEST_PATH"); exists {
+		vendorDir = filepath.Join(workingDir, destPath)
+	}
+
 	var args []string
-	_, err := os.Stat(filepath.Join(workingDir, "vendor"))
-	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			args = []string{
-				"install",
-				"--requirement",
-				"requirements.txt",
-				"--exists-action=w",
-				fmt.Sprintf("--cache-dir=%s", cachePath),
-				"--compile",
-				"--user",
-				"--disable-pip-version-check",
-			}
-		} else {
-			return err
-		}
+	_, err := os.Stat(vendorDir)
```

Comment:
I think this can be simplified via use of the [`fs.Exists()` helper function in packit](https://github.com/paketo-buildpacks/packit/blob/6954a424c44bf1724da5f70e7fe1dc391f500e78/fs/exists.go).

Reply:
Good call. Will make that change.